S/SatelliteToolbox/Compat.toml

```diff
@@ -38,6 +38,12 @@ Crayons = "4.0"
 Interpolations = "0.11.2"
 OptionalData = "0.2.1"
 Parameters = "0.10.3"
-ReferenceFrameRotations = "0.5.1"
 StaticArrays = "0.10.3"
+
+["0.6.0"]
+ReferenceFrameRotations = "0.5.1"
 julia = "1.0-1.3"
+
+["0.6.1"]
+ReferenceFrameRotations = "0.5.1-0.5.2"
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.